### ISSUE TITLE
display of `<cei:nota>` and `<cei:rubrum>` in preview is odd

### ISSUE THREAD

**Issue by GVogeler** [icaruseu/mom-ca#771]
when creating several `<cei:nota>` and `<cei:rubrum>` elements each would represent one note by the chancery / archive. In the preview of the single charter (and probably in single charter view in general) the entries are listed just one after the other:

![grafik](https://user-images.githubusercontent.com/324006/40830795-997177f6-6587-11e8-908f-a61e191b6f5a.png)

leeds to 

![grafik](https://user-images.githubusercontent.com/324006/40830818-a7d547dc-6587-11e8-9732-603395d290fe.png)

**Comment by GVogeler** [comment]
https://github.com/icaruseu/mom-ca/blob/master/my/XRX/src/mom/app/charter/xsl/cei2html.xsl#L1078-1080 is just iterating through the elements without any list distinguishing elements. Maybe the better solution would be replating the `for-each` with an `<ul><apply-templates selecte="cei:rubrum"/></ul>` and a dedicated template, e.g.
```
<xsl:template match="cei:rubrum">
       <li><xsl:apply-templates/></li>
 </xsl:template>
```

**Comment by GVogeler** [comment]
the same is valid for https://github.com/icaruseu/mom-ca/blob/master/my/XRX/src/mom/app/charter/xsl/cei2html.xsl#L1064-L1066

**Comment by GVogeler** [comment]
solved with #772
